pennylane_lightning/lightning_gpu/_adjoint_jacobian.py

```diff
@@ -15,10 +15,11 @@
 Internal methods for adjoint Jacobian differentiation method.
 """
 
+from __future__ import annotations
```

Comment:
Added `future` to improve the typing for `LightningGPUStateVector` without loading the module again.